docker-compose.yml

```diff
@@ -0,0 +1,12 @@
+version: "3"
+services:
+  starknet-node:
+    image: eqlabs/pathfinder
+    restart: unless-stopped
+    environment:
+      - RUST_LOG=info
+      - PATHFINDER_ETHEREUM_API_URL=https://mainnet.infura.io/v3/dd4eaec0a4b745f2b1aa21378d5bf044
```

Comment:
I don't think including your secrets (like your private Infura URL) in a public repository is a good idea. Can't we just remove this and still provide it as an external environment variable?

Reply:
@sven-hash I would rebase / squash that commit away so that the url isn't scrapable by bots on github :)